### ISSUE TITLE
brm_multiple with cmdstanr backend prints all progress after all models are run

### ISSUE THREAD

**Issue by StaffanBetner** [paul-buerkner/brms#1059]
When I use brm_multiple with the cmdstanr backend all progress printing is made after all models have run, which makes it harder to keep track of the model fitting process. I am not sure whether it is an issue with brms or cmdstanr. I am running Windows, which might be relevant.

**Comment by paul-buerkner** [comment]
Yeah, that is a current behavior of using `future` for running these models (even in serial). Here is an explanation of how future handles output: https://cran.r-project.org/web/packages/future/vignettes/future-2-output.html
I don't think there is anything I can do about it in brms unfortunately. But please feel free to reopen if you have a have an idea for how to fix it from the brms side.

**Comment by StaffanBetner** [comment]
Ah, so it's due to `future`! Good to know!
